lib/field_test/participant.rb

```diff
@@ -29,7 +29,7 @@ def where_values
         }
       else
         {
-          participant_type: type,
+          participant_type: type ? type.constantize.base_class.to_s : type,
```

Comment:
I see you are testing an empty string as the type. In that case, won't this blow up?